src/common/options/ceph-exporter.yaml.in

```diff
@@ -52,3 +52,12 @@ options:
   - ceph-exporter
   flags:
   - runtime
+- name: exporter_get_labeled_counters
+  type: bool
+  level: advanced
+  desc: If true will fetch labeled perfcounters from counter dump command
+  default: false
```

Comment:
For the  `long_desc` I think we already have docs for perf counters updated that they support the labeled counters as well. So probably just a short description should be fine as well, no?

Reply:
And it's option to **fetch** the labeled perf counters not enable, because they already exists in `counter dump` command if any labeled counters are added.

Reply:
And yes I agree that we should have this option set to `true` by default. The user can set it to false in case they don't need them.